Classes/Controllers/PBGitHistoryController.m

```diff
@@ -902,6 +905,45 @@ - (IBAction)showTagInfoSheet:(id)sender
 	[self.windowController showMessageSheet:title infoText:info];
 }
 
+
+- (void)didDoubleClickCommit:(id)sender
+{
+	id refish = nil;
+	
+	// If they double-clicked row 1 (the one with ref labels in) we need to check
+	// if they clicked a ref label.
+	if (commitList.clickedColumn == 1)
```

Comment:
Columns can be rearranged, so the subject column isn't always at index 1. Try this instead:

```objc
if ([commitList.tableColumns[clickedColumn].identifier isEqualToString:@"SubjectColumn"])
```

Reply:
Nicely spotted - didn't know you could rearrange them. I tested your code and it works perfectly.